requirements.txt

```diff
@@ -1,8 +1,8 @@
-numpy==1.25.1
-pandas==2.0.3
-pyarrow==12.0.*
+numpy==1.26.*
+pandas==2.*
+pyarrow==13.0.*
```

Comment:
Bumping these to any minor version. We'll need to manually bump some of them to get the correct wheels for `3.12`.